### DISCUSSION TITLE
OLEDモジュールのジャンパ部分ランド剥離への対応について

### DISCUSSION THREAD

**Discussion by 2000ap2** [Yowkees/keyball#448] [Q&A] [answered]
お世話になっております。下記について対処方法をご教授いただきたく投稿させていただきます。

Keyball39トラックボールなし側基盤の裏側にて、図中のショートさせるランド（赤枠線部）を剥離させてしまいました。
可能であれば、被覆線による接続での対応を考えております。
繋ぐべき２点をご教示いただきたいです。

お忙しいところ恐れ入りますが以上よろしくお願いいたします。

![20231213_002548](https://github.com/Yowkees/keyball/assets/41547814/0f0f8f80-3cde-4cec-8a34-d473cab28b8e)

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

写真に示してあります赤点どうしを繋いでいただきますとジャンパ部分のはんだ付けをしたことになりますので、ご対応のほどよろしくお願いいたします。
![289921313-0f0f8f80-3cde-4cec-8a34-d473cab28b8e](https://github.com/Yowkees/keyball/assets/143698349/fe77a2f8-e09c-4b25-940e-ffe0bcc9df6b)

**Reply by 2000ap2** [reply]
ご教示いただきありがとうございます。
２点を線で繋いだ状態で動作確認したところ、不具合は見当たらず正常に動作しております。

迅速なご回答をいただきありがとうございました。

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
ぜひ末永くご愛用していただければ幸いです。
